tests/upload_tlc_stats.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import json
+import os
+import argparse
+import cimetrics.upload
+
+from loguru import logger as LOG
+
+
+def run(filename):
+    if os.path.exists(filename):
+        with open(filename) as f:
+            data = json.load(f)
+            duration_sec = data["duration"]
+            dstates = data["distinct"]
+            LOG.info(
+                "Uploading metrics - duration: {}, distinct states: {}",
+                duration_sec,
+                dstates,
+            )
+            with cimetrics.upload.metrics(complete=False) as metrics:
```

Comment:
I don't know what cimetrics provides but https://github.com/tlaplus/Examples/blob/62d104f2e1cfd63b3f928342b8f634298706d145/specifications/ewd998/EWD998ChanID_export.tla#L46-L56 is an example about how you can do post requests from within TLC, which would make the json serialization obsolete.

Reply:
This is cimetric https://github.com/jumaffre/cimetrics, it provides the fancy performance graphs in each PR. 

I think the Json stats file might be useful in the future to quickly compare the performance of different configurations

Reply:
If more is going on in cimetrics than just a basic http post, the .py script is the way to go. :-)

Btw. I do have the CosmosDB client library wrapped as a TLA+ module somewhere, but that's just an interesting hack.

Reply:
Upload is pretty straightforward: https://github.com/jumaffre/cimetrics/blob/main/cimetrics/upload.py, most of the logic is there to the querying and the fancy plotting (which you can see in this PR too if you scroll up a bit).

For procedural tasks such as this one, Python has a certain charm :)